docs/en/setup/backend/backend-storage.md

```diff
@@ -31,14 +32,20 @@ storage:
     user: sa
 ```
 
+## OpenSearch
+
+OpenSearch storage shares the same configurations as ElasticSearch 7.
+In order to activate ElasticSearch 7 as storage, set storage provider to **elasticsearch7**.
+Please download the `apache-skywalking-bin-es7.tar.gz` if you want to use OpenSearch as storage.
+
 ## ElasticSearch
 - In order to activate ElasticSearch 6 as storage, set storage provider to **elasticsearch**
 - In order to activate ElasticSearch 7 as storage, set storage provider to **elasticsearch7**
 
 **Required ElasticSearch 6.3.2 or higher. HTTP RestHighLevelClient is used to connect server.**
 
```

Comment:
We need a NOTICE here to indicate from which versions, ElasticSearch became to be licensed by SSPL, which is not compatible with Apache 2.0 anymore.

> NOTICE: MySQL driver is NOT allowed in Apache official distribution and source codes. Please download MySQL driver by yourself. Copy the connection driver jar to oap-libs.

Reply:
We don't know yet from which version, do we? Also, we're still using APLv2 licensed jars, why bother telling users to download by themselves before we have to remove ES client jars in our distribution?

Reply:
The client API jar's license is not all. Because the dependency is checked from the runtime. We should say after which versions(released), they changed LICENSE. No matter Elastic called these new versions, they are already making us to depend on SSPL licensed server.

Reply:
I just don’t see the point to add the notice to the users because the clients jar we are using now can be safely used no matter what server version the users use. And if the users want to download more recent versions (SSPL), that’s Elastic’s or / and users’ duty to check their license issue. The reason why MySQL jar needs a notice is that IMO it needs users action explicitly otherwise our project won’t work when using MYSQL as storage.

Reply:
We as ASF TLP, definitely have obligation to tell end users, there is License incompatible risk.
The foundation requires us for public good, and our release should make sure of it. Apache license v2 is one of the most important things as the foundation.

Reply:
We write the source codes to tell users, we recommend Elasticsearch as the storage option. We shouldn't say this is your duty about legal or license usage.

Reply:
I wouldn't say SSPL is a bad thing, I just argue from the ASF perspective.